tests/test_api.py

```diff
@@ -10,9 +12,14 @@
 
 from damnit import Damnit, RunVariables
 from damnit.context import ContextFile
+from damnit.backend.user_variables import UserEditableVariable
 from .helpers import extract_mock_run
 
 
+# We disable updates from VariableData.write() by default so we can use the
+# convenient RunVariables.__setitem__() method.
+os.environ["DAMNIT_SEND_UPDATE"] = "0"
```

Comment:
It might be good to put this in an [autouse fixture](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request), rather than doing it as a side effect of importing the module. Side effects on imports can be surprising.

Unless we're setting the environment variable for the whole test suite, it would also be a good idea to use monkeypatch.setenv so it gets changed back after the tests it affects.

Reply:
Excellent point, switched to using monkeypatch in 966739c.